sklearn/linear_model/_base.py

```diff
@@ -472,11 +473,12 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
     """
     @_deprecate_positional_args
     def __init__(self, *, fit_intercept=True, normalize=False, copy_X=True,
-                 n_jobs=None):
+                 n_jobs=None, solver="auto"):
```

Comment:
Please update the docstring above to describe this parameter and how it works. Not sure if we need "auto" here or just a choice between two explicit solvers

Reply:
I think in this PR we should do just the two solvers, and then potentially have another PR with benchmarks and a heuristic.